src/dom.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable no-unused-vars */
+
+let isCel = true;
+let deg;
+let celDeg;
+let ferDeg;
+
+
+export default function render(data) {
+  const card = document.getElementById('detailsCard');
+  card.classList = 'card bg-secondary text-white mx-auto my-3';
+  const {
+    main, name, sys, weather, ...rest
+  } = data;
+
+  const cel = +(`${Math.round(`${main.temp - 273.15}e+2`)}e-2`);
+  celDeg = `${cel} °C `;
+  const fer = +(`${Math.round(`${(main.temp - 273.15) * 1.8000 + 32.00}e+2`)}e-2`);
+  ferDeg = `${fer} °F `;
```

Comment:
This is also a logic that can be abstracted away from the dom, this is ideally a utility function

Reply:
@Oghenebrume50   Agreed. but this could be mentioned as optional changes too :|

Reply:
this function could be declared in other file, but this will make redundant use of folllowing local variables. so skipping this .
let isCel = true;
let deg;
let celDeg;
let ferDeg;